dataclients/kubernetes/testdata/ingressV1/ingress-data/ing-with-label-selector-filter-defaults-to-ingress.eskip

```diff
@@ -0,0 +1,4 @@
+kube_foo__qux__www_example_org_____bar:
+  Host("^(www[.]example[.]org[.]?(:[0-9]+)?)$") &&
+  PathRegexp("^/")
```

Comment:
Predicates in one line

Reply:
Done!

I think I copied it from [another file](https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/testdata/ingressV1/ingress-data/ing-with-setpath-filter-annotation.eskip) and just followed the same style as found there.

I can change it everywhere in this PR if you ask me to, that's easy :)